docs/whirlpool/docs/02-Architecture Overview/07-TokenExtensions Support.md

```diff
@@ -0,0 +1,161 @@
+# TokenExtensions
+
+At slot `268396603` (May 28, 2024 07:08:16 UTC), Whirlpool program have been upgraded to support TokenExtensions (aka Token-2022 program).
```

Comment:
The slot is only relevant for Solana. Should we just remove it and only add the date?